larndsim/consts.py

```diff
@@ -112,6 +120,7 @@ def load_detector_properties(detprop_file, pixel_file):
     global ys
     global pixel_pitch
     global tpc_borders
+    global tpc_offsets
```

Comment:
Why are you adding this global variable here? It looks like it's not used anywhere.

Reply:
this was added while we were reworking the voxelization function.  That function now references `tpc_borders`, which is already global, so this is reverted in https://github.com/DUNE/larnd-sim/pull/39/commits/2726498f5bd170582e856636bb0d4f138a4ec047.  Good catch!

Reply:
We got that fixed in [`2726498`](https://github.com/DUNE/larnd-sim/pull/39/commits/2726498f5bd170582e856636bb0d4f138a4ec047)